wagtail/models/__init__.py

```diff
@@ -2699,6 +2747,95 @@ class Meta:
         ]
 
 
+class CustomPageManager(models.Manager):
+    @transaction.atomic
+    def bulk_create(self, objs, **kwargs):
+        pages_info = {}
+        for page in objs:
+            page_post_details = {}
+            if kwargs.get("clean", True):
+                page.full_clean()
+
+            page_post_details["slug_changed"] = False
```

Comment:
bulk_create() should only be uses to create new items, so this shouldn't be needed (We shouldn't ever modify supplied slugs either)